### PR TITLE
HOTFIX: make sure all ConsumerGroupServices get closed

### PR DESCRIPTION
**Pull request by chia7712**
Some services are not closed, so our CI print following error.
```
org.opentest4j.AssertionFailedError: Found 16 unexpected threads during @BeforeAll: `kafka-admin-client-thread | adminclient-287,kafka-admin-client-thread | adminclient-276,kafka-admin-client-thread | adminclient-271,kafka-admin-client-thread | adminclient-293,kafka-admin-client-thread | adminclient-281,kafka-admin-client-thread | adminclient-302,kafka-admin-client-thread | adminclient-334,kafka-admin-client-thread | adminclient-323,kafka-admin-client-thread | adminclient-257,kafka-admin-client-thread | adminclient-336,kafka-admin-client-thread | adminclient-308,kafka-admin-client-thread | adminclient-263,kafka-admin-client-thread | adminclient-273,kafka-admin-client-thread | adminclient-278,kafka-admin-client-thread | adminclient-283,kafka-admin-client-thread | adminclient-317` ==> expected: <true> but was: <false>
```
#15679 use `AfterEach` to release service. However, the test cases having multi consumerConfigs will create a lot of services in testing. Hence, the intermediate servers are not closed. 

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by FrankYang0529** [comment]
@chia7712, thanks for the hotfix. I'm very sorry for the inconvenient. The fix LGTM. It's better to use try-with-resource to close resource. Thanks.

File: tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java [hunk 1]
```diff
@@ -171,8 +162,8 @@ private String[] getArgs(String group, String topic) {
         };
     }
 
-    private void setupConsumerGroupService(String[] args) {
-        consumerGroupService = new ConsumerGroupCommand.ConsumerGroupService(
+    private static ConsumerGroupCommand.ConsumerGroupService consumerGroupService(String[] args) {
```
Comment (showuon) [inline tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java:165]: Why should we use static method here?

**Review by showuon — APPROVED** [review]
LGTM! Just one minor question, but it can be addressed after fixing the CI build. Thanks.

**Comment by showuon** [comment]
The thread leaking failed tests disappeared now.

**Pull request merged**